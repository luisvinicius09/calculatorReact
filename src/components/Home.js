import React from 'react';
import NavBar from './Navbar';

const Home = () => (
  <>
    <NavBar />
    <h1>Hello from the real world 2</h1>
  </>
);

export default Home;
