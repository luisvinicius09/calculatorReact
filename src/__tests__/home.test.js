import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Home from '../components/Home';

it('renders the homepage correctly', () => {
  const home = renderer.create(
    <BrowserRouter>
      <Home />
    </BrowserRouter>,
  ).toJSON();
  expect(home).toMatchSnapshot();
});
