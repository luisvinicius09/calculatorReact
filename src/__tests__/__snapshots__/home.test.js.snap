// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the homepage correctly 1`] = `
Array [
  <div
    style={
      Object {
        "display": "flex",
        "flexDirection": "row",
        "justifyContent": "space-between",
      }
    }
  >
    <div
      style={
        Object {
          "marginLeft": "10px",
        }
      }
    >
      <h1>
        Math Magicians
      </h1>
    </div>
    <div
      style={
        Object {
          "margin": "30px",
        }
      }
    >
      <a
        href="/"
        onClick={[Function]}
        style={
          Object {
            "paddingRight": "10px",
          }
        }
      >
        Home
      </a>
      <a
        href="/calculator"
        onClick={[Function]}
        style={
          Object {
            "paddingRight": "10px",
          }
        }
      >
        Calculator
      </a>
      <a
        href="/quote"
        onClick={[Function]}
      >
        Quote of the day
      </a>
    </div>
  </div>,
  <div
    style={
      Object {
        "margin": "50px",
      }
    }
  >
    <h1>
      Home
    </h1>
    <p>
      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
    </p>
  </div>,
]
`;
